client/src/components/geneExpression/infoPanel/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { connect } from "react-redux";
+import { AnchorButton, ButtonGroup } from "@blueprintjs/core";
+import { AppDispatch, RootState } from "../../../reducers";
+import {
+  CollapseToggle,
+  InfoPanelContent,
+  InfoPanelHeader,
+  InfoPanelTabs,
+  InfoPanelWrapper,
+  StyledAnchorButton,
+} from "./style";
+import GeneInfo from "./geneInfo";
+import DatasetInfo from "./datasetInfo";
+
+interface Props {
+  activeTab: string;
+  dispatch: AppDispatch;
+  infoPanelMinimized: boolean;
+  infoPanelHidden: boolean;
+}
+
+const InfoPanel = (props: Props) => {
```

Comment:
super nit: Can we change this from function expression to function declaration please 🙏 ? I believe stack tracing works a lot better with declaration than an anonymous function!

Here and throughout

E.g.,

```ts
function InfoPanel(props: Props) {
  ...
}
```